### ISSUE TITLE
torchvision.io.read_image() fails to read image

### ISSUE THREAD

**Issue by drdebmath** [pytorch/vision#4738]
### 🐛 Describe the bug

```
import torch
import torchvision.io as io
image = io.read_image('./1.png')
```

This minimal example does not work. I have tried many different versions of torch with different versions of python.
It was working previously, and now it fails after doing the update.

Some of the errors that I got are:
RuntimeError: No such operator image::read_file 

I also tried the same in python3.7.11 and torchvision 0.8
and got RuntimeError: Non RGB images are not supported.  

### Versions

Collecting environment information...
PyTorch version: 1.10.0
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: macOS 11.6 (x86_64)
GCC version: Could not collect
Clang version: 13.0.0 (clang-1300.0.29.3)
CMake version: Could not collect
Libc version: N/A

Python version: 3.8.8 (default, Apr 13 2021, 12:59:45)  [Clang 10.0.0 ] (64-bit runtime)
Python platform: macOS-10.16-x86_64-i386-64bit
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] mypy-extensions==0.4.3
[pip3] numpy==1.20.1
[pip3] numpydoc==1.1.0
[pip3] torch==1.10.0
[pip3] torchaudio==0.10.0
[pip3] torchvision==0.11.1
[conda] blas                      1.0                         mkl  
[conda] ffmpeg                    4.3                  h0a44026_0    pytorch
[conda] mkl                       2021.2.0           hecd8cb5_269  
[conda] mkl-service               2.3.0            py38h9ed2024_1  
[conda] mkl_fft                   1.3.0            py38h4a7008c_2  
[conda] mkl_random                1.2.1            py38hb2f4e1b_2  
[conda] mypy_extensions           0.4.3                    py38_0  
[conda] numpy                     1.20.1           py38hd6e1bb9_0  
[conda] numpy-base                1.20.1           py38h585ceec_0  
[conda] numpydoc                  1.1.0              pyhd3eb1b0_1  
[conda] pytorch                   1.10.0                  py3.8_0    pytorch
[conda] torchaudio                0.10.0                 py38_cpu    pytorch
[conda] torchvision               0.11.1                 py38_cpu    pytorch

**Comment by ngam** [comment]
@NicolasHug 

> @malfet do we not support the latest torchvision for M1 archs?

No you don't, the latest available torchvision for M1 from -c pytorch is 0.2.2. I have filed an issue for this same problem in conda-forge repo (see https://github.com/conda-forge/torchvision-feedstock/issues/32). Interestingly, one will get this error when using the latest NVIDIA pytorch container because... well... it relies on the conda-forge packages!

**Comment by h-vetinari** [comment]
This looks like a reappearance of https://github.com/pytorch/vision/issues/4181

**Comment by malfet** [comment]
@NicolasHug we don't build publish M1 binaries to conda, although we probably should

**Comment by ngam** [comment]
@NicolasHug + @malfet --- 

Are there underlying issues preventing you from building for ARM? 

Even the conda-forge builds for Linux are not working... The build by conda-forge team (@h-vetinari is here; conda-forge usually includes builds for all OSs) is still failing for some reason. It would be good to resolve this since a lot of people (including the NVIDIA containers!) rely on the -c conda-forge builds and not on the -c pytorch builds. 

@malfet could you please have a look (https://github.com/conda-forge/torchvision-feedstock/pull/31) to help the team resolve this soon? Also, I've been browsing all the issues around this to try to see what I could do to help and I realize that torchaudio isn't supported on M1. As an exercise, I started a build for torchaudio on the conda-forge side (https://github.com/conda-forge/staged-recipes/pull/16720) so if you have time, I would love your insights there as well. I am major newbie here, but I am happy to learn and help as much as I could :) 

Thanks all for all your work --- it really makes a difference to academic researchers like me, and it inspires me to do more to help 🥇

**Comment by cainiaoup** [comment]
you can use opencv to load your image .

1. pip install opencv-python
2. let cv.imread(img_path)  instand of read_image(img_path)

**Comment by ngam** [comment]
This issue has been resolved in 0.11.2, I believe.
